app/auth/routes.py

```diff
@@ -5,6 +5,9 @@
 from app.auth import bp
 from app.auth.forms import LoginForm, RegistrationForm, ResetPasswordRequestForm
 from app.models import User
+from app.emails.email import send_password_reset_email
+from app.auth.forms import ResetPasswordForm
```

Comment:
The ResetPasswordForm can be added to the other imported forms above.

Reply:
done